doi.jq

```diff
@@ -0,0 +1,186 @@
+		"jobber",
+		"mageia",
+		"mono",
+		"nats",
+		"nats-streaming",
+		"oraclelinux", # direct children: mysql openjdk percona
+		"percona",
+		"photon",
+		"php", # direct children: backdrop composer drupal friendica joomla matomo mediawiki monica nextcloud phpmyadmin postfixadmin unit wordpress yourls
+		"php-zendserver",
+		"phpmyadmin",
+		"postfixadmin",
+		"r-base",
+		"rocket.chat",
+		"rockylinux",
+		"sl",
+		"spiped",
+		"teamspeak",
+		"traefik",
+		"ubuntu", # direct children: buildpack-deps couchbase eclipse-temurin elasticsearch gazebo gradle ibmjava ibm-semeru-runtimes kibana kong logstash mariadb mongo neurodebian odoo php-zendserver rabbitmq ros sapmachine silverpeas swift
+		"varnish",
+		"vault",
+		"yourls",
+		"znc",
+		"adminer",
+		"aerospike",
+		"amazoncorretto", # direct children: jetty maven tomcat
+		"backdrop",
+		"buildpack-deps", # direct children: erlang gcc golang haskell haxe influxdb kapacitor node openjdk perl pypy python rakudo-star ruby rust telegraf
+		"composer", # direct children: drupal
+		"couchbase",
+		"dart",
+		"eclipse-temurin", # direct children: cassandra clojure flink gradle groovy jetty jruby lightstreamer liquibase maven neo4j orientdb solr sonarqube spark storm tomcat tomee unit zookeeper
+		"erlang", # direct children: elixir
+		"friendica",
+		"gazebo",
+		"groovy",
+		"haskell",
+		"haxe",
+		"ibm-semeru-runtimes", # direct children: maven open-liberty tomee websphere-liberty
+		"ibmjava", # direct children: maven websphere-liberty
+		"jetty", # direct children: geonetwork
+		"joomla",
+		"kong",
+		"lightstreamer",
+		"liquibase",
+		"matomo",
+		"mediawiki",
+		"monica",
+		"neurodebian",
+		"nextcloud",
+		"node", # direct children: express-gateway ghost mongo-express unit
+		"odoo",
+		"open-liberty",
+		"orientdb",
+		"python", # direct children: hylang plone satosa unit
+		"rakudo-star",
+		"ros",
+		"sapmachine", # direct children: maven
+		"satosa",
+		"silverpeas",
+		"spark",
+		"storm",
+		"swift",
+		"tomcat", # direct children: convertigo geonetwork xwiki
+		"tomee",
+		"websphere-liberty",
+		"clojure",
+		"convertigo",
+		"geonetwork",
+		"maven",
+		"plone",
+
+		empty
+	]
+;
+
+# a helper for "build_should_sbom"
+def _sbom_subset:
+	[
+		# only repositories we have explicitly verified
+		"bash",
+		"buildpack-deps",
+		"busybox",
+		"caddy",
+		"cassandra",
+		"chronograf",
+		"couchdb",
+		"drupal",
+		"eclipse-temurin",
+		"elasticsearch",
+		"flink",
+		"fluentd",
+		"gcc",
+		"ghost",
+		"gradle",
+		"haproxy",
+		"httpd",
+		"hylang",
+		"influxdb",
+		"jruby",
+		"julia",
+		"kapacitor",
+		"kibana",
+		"logstash",
+		"mariadb",
+		"memcached",
+		"mongo",
+		"mongo-express",
+		"mysql",
+		"neo4j",
+		"nginx",
+		"openjdk",
+		"perl",
+		"php",
+		"postgres",
+		"pypy",
+		"python",
+		"rabbitmq",
+		"redis",
+		"registry",
+		"rethinkdb",
+		"ruby",
+		"rust",
+		"solr",
+		"sonarqube",
+		"telegraf",
+		"tomcat",
+		"wordpress",
+		"xwiki",
+		"zookeeper",
```

Comment:
This list has 50 elements. The tracking spreadsheet has 45. @LaurentGoderre can we explain the difference?

Reply:
The meta column of the list is at 50. Some images were migrated but don't have SBOM yet or don't have accurate ones (earlier ones that we merged before we haver the current process).

Reply:
But this is the sbom list, not meta list, right?

Reply:
I stole this list from the spreadsheet to begin with, and then added to it the entries from the open PRs :see_no_evil:

IIRC, it was 45 from the sheet and then 6 open PRs which must've had some overlap -- maybe the sheet wasn't updated with all the new open PRs?

Reply:
(mostly an easy fixup PR if I've mucked it up, though! :smile:)